index.html

```diff
@@ -294,90 +291,6 @@ <h3>
           </li>
         </ol>
       </section>
-      <section>
-        <h3>
-          <dfn>onvisibilitychange</dfn> event handler attribute
-        </h3>
-        <p>
-          The {{Document/onvisibilitychange}} attribute is an <a>event handler
-          IDL attribute</a> for the `"visibilitychange"` event type (see
-          [[[#reacting-to-visibilitychange]]]).
-        </p>
-      </section>
-    </section>
-    <section>
-      <h2 id="reacting-to-visibilitychange">
-        Reacting to `"visibilitychange"` changes
-      </h2>
-      <p>
-        The <a>task source</a> for these <a>tasks</a> is the <a>user
-        interaction task source</a>.
-      </p>
-      <p>
-        When the user agent determines that the visibility of the {{Document}}
-        of the <a>top-level browsing context</a> has changed, the user agent
-        MUST run the following steps:
-      </p>
-      <ol class="algorithm">
-        <li>Let |doc:Document| be the {{Document}} of the <a>top-level browsing
-        context</a>.
-        </li>
-        <li>If |doc| is now [=Document/visible=]:
-          <ol>
-            <li>If traversing to a <a>session history entry</a>, run the <a>now
-            visible algorithm</a> before running the step to fire the
-            [=Window/pageshow=] event.
-            </li>
-            <li>Otherwise, <a>queue a task</a> that runs the <a>now visible
```

Comment:
The events defined in HTML now seem to be run synchronously, which is not what was previously defined. Is this intentional? Do we know what implementations are doing here?

Reply:
The HTML spec only defines the event order in the cases of unload/page hide. Those have to be synchronous otherwise they will never be handled 

What's not defined is the non standard firing of that event